### PR TITLE
Test milestone 1

### PR DESCRIPTION
**Pull request by olawale-o**
## The following was implemented:
- Mock localStorage (__mocks__/mockStorage.js)
  -- Add to localStorage 
  -- Remove from localStorage
- Mock DOM (__tests__/dom.test.js)
 -- Add new task to the DOM
-- Remove task from the DOM

### CODE REVIEW

File: __tests__/add-remove.test.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+  describe('Delete Todo', () => {
+    test('task 1 should be equal {description; task 1, completed: false, index: 1} ', () => {
+      let todos = [];
+      const expectedTodo = { description: 'task 1', completed: false, index: 1 };
+
+      todos = onDeleteTodo(expectedTodo.id, todos);
+
+      expect(todos.length).toEqual(0);
+    });
```
Comment (Dipeshtwis) [inline __tests__/add-remove.test.js]: - [x] I would recommend to add more test cases for `add` and `delete` todo's so that it will cover all the edge test case as well, like ` where there is more than 2 todo's and deleting 1 will delete only 1 and rest will remain there.`

File: __tests__/add-remove.test.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      const task = 'task 1';
+      const todos = [];
+      const expectedTodo = { description: 'task 1', completed: false, index: 1 };
+      const newTodo = addNewTodo(task, todos);
+      expect(newTodo).toEqual(expectedTodo);
+    });
+  });
+
+  describe('Delete Todo', () => {
+    test('task 1 should be equal {description; task 1, completed: false, index: 1} ', () => {
```
Comment (Dipeshtwis) [inline __tests__/add-remove.test.js]: - [x] I think this `delete todo` test description does not suits well. It supposed to be like `todo will reduced to 0 if we have one task earlier.`

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @olawale-o and @Abdona, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: It has a test file (`[..].test.js`) for a file containing the **add item** and **delete item** functions that would be tested.
:+1: Uses mock for funtions that used `localStorage` , `DOM`.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
## Hi @olawale-o 👋🏾 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
